SoftwareDesign.md

```diff
@@ -0,0 +1,16 @@
+*Favorite Design Pattern: Model View Controller*
+
+
+I really enjoy the MVC design pattern, probably because I have the most familiarity with it. I have used it before with Full Stack Javascript Code, and I am gaining familiarity with using it while coding in Python and Django. 
+
+
+I'm reading about patterns more, as another Dev I met with reccommended the same book that Aaron did: Design Patterns: Elements of Reusable Object-Oriented Software. I had actually purchased it a few weeks ago but hadn't cracked it open yet.
+
+
+I'm reading through it now and trying to absorb as much as possible.
+
+
+*Favorite Design Anti-Pattern: WET (Write Every Time) Code*
+
+
+I find myself doing this far too much. In fact, the less I plan my code out ahead of time, the more repetitive I am. If I plan ahead, I can make a plan to write reusable code and that generally helps me a lot when it comes to writing DRY code.
```

Comment:
these are things that come w/ time.  You are awesome and are doing great.  Planning takes foresight and foresight takes experience.  You'll be killing it and will be doing so less repetitively soon : )